nixos/modules/services/monitoring/netdata.nix

```diff
@@ -9,10 +9,21 @@ let
     mkdir -p $out/libexec/netdata/plugins.d
     ln -s /run/wrappers/bin/apps.plugin $out/libexec/netdata/plugins.d/apps.plugin
   '';
-
+  configDirectory = pkgs.runCommand "netdata-config-d" {} ''
+    mkdir -p $out
+    cp -r ${pkgs.netdata}/etc/netdata/* $out
```

Comment:
I believe this is supposed to be `${pkgs.netdata}/lib/netdata/conf.d/*`

Reply:
No, it's not. The health alarm configuration is not stored inside `conf.d` but next it.

Reply:
Oh, it's probably because I'm on NixOS 19.03, netdata is at version 1.11, apologies